addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-rstudio-instance.cfn.yml

```diff
@@ -217,9 +217,6 @@ Outputs:
   #  Currently this is applicable only when ConnectionScheme = 'ssh'.
   #  This instanceId will be used for sending user's SSH public key using AWS EC2 Instance Connect when user wants to SSH to the instance.
   #
-  MetaConnection1Info:
```

Comment:
Is there a reason it needs to be deleted? Also the comment above still mentions MetaConnection1Info.

Reply:
Why deleted - Redundant. The OutputValue for it was the same as the one for 'Ec2WorkspaceDnsName', and nowhere else in the code was it used anyways.
Comment - The comment block for all SC product templates is the same.